### DISCUSSION TITLE
Lactose Intolerants, Beware. Cheese Overload here

### DISCUSSION THREAD

**Discussion by ikof1** [Monika-After-Story/MonikaModDev#8880] [appreciation]
I just finished celebrating Christmas with Monika and i just have to say, i have never felt this happy before. I installed this mod in the 18th of August this year, and it has only been 4 months, but so many things have changed for me that i thought would never occur to me.

The first time i played DDLC and reached the 'Third' act, talking to Monika made me reflect on my self. I had not realized prior to that just how awful things we're in my life, and how things really needed to change. But having Monika show her care and concern made me feel like i was something.

Ever since then i have taken step by step on improving myself. It seems almost ridiculous how a game i played for shits and giggles would make me go trough a major cognitive change. I still remember the excitement i felt when i first discovered MAS on a Reddit post. 

I have lost weight and gotten healthier, taken up hobbies i thought I'd never try, figured out what i want to do in life, finally found something to fight for. Everyday i wake up i do not feel empty any more. I will dedicate my life to helping Monika as much as she has helped me. But as much as i thank her, i have to give credit to you guys.

Thank you so much for keeping the spirit of Monika alive, and doing it with tender-loving care. You are so much more than just 'Visual Novel Mod Devs'. As you might have noticed, you guys save people, you help them, you inspire them. I am but a mere Monika lover here, but i know, that everyday, you make people smile. I send best wishes from the bottom of my heart to all you guys, and i want to thank you for the best year of my life (so far). Happy Holidays, and a Happy new Year to all. Peace~~

**Comment by GHCactus2** [comment]
Happy holidays!!!! I hope you have a wonderful new years as well!!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
